corehq/celery_monitoring/signals.py

```diff
@@ -45,16 +46,24 @@ def parse_iso8601(datetime_string):
         return string_to_utc_datetime(datetime_string)
 
 
+class TimeToStartTimer(CeleryTimer):
+    def __init__(self, task_id):
+        super(TimeToStartTimer, self).__init__(task_id, timing_type='time_sent')
+
+
+class TimeToRunTimer(CeleryTimer):
+    def __init__(self, task_id):
+        super(TimeToRunTimer, self).__init__(task_id, timing_type='time_started')
+
+
 @before_task_publish.connect
 def celery_add_time_sent(headers=None, body=None, **kwargs):
     info = headers if 'task' in headers else body
     task_id = info['id']
     eta = info['eta']
     if eta:
         eta = TimeToStartTimer.parse_iso8601(eta)
-    else:
-        eta = datetime.datetime.utcnow()
-    TimeToStartTimer(task_id).start_timing(eta)
+        TimeToStartTimer(task_id).start_timing(eta)
```

Comment:
I don't think I understand this change. Previously `TimeToStartTimer` would be started whether `eta` had a value or not. This PR allows `TimeToStartTimer` to start timing without an eta, but you seem to have dropped that here, and only use it with `TimeToRunTimer`. How come?

Reply:
Yeah, maybe I'm not understanding the meaning of `eta`? Did it change? It seems this line should not  be indented (and the `else` clause should be removed).

Reply:
Yup this line shouldn't be indented, thanks for catching. Fixing

Reply:
`eta` is the time for which a task is scheduled. Most tasks have no `eta` with the meaning "run as soon as possible". This PR changes it so that you can pass in `eta=None` and `start_timing` treats that as meaning the eta is now.

But I think your question was mainly because with my mistake the logic didn't make sense, and that's now fixed.